src/output-json-alert.c

```diff
@@ -295,8 +295,9 @@ static void AlertAddAppLayer(const Packet *p, JsonBuilder *jb,
     const AppProto proto = FlowGetAppProtocol(p->flow);
     EveJsonSimpleAppLayerLogger *al = SCEveJsonSimpleGetLogger(proto);
     JsonBuilderMark mark = { 0, 0, 0 };
+    void *state;
```

Comment:
why did this get changed too?

Reply:
Eheh.. Because you cannot declare variables just after a case of a switch

> output-json-alert.c:389:13: error: expected expression
  389 |             void *state = FlowGetAppState(p->flow);

I guess that is why I had begun by `jb_get_mark`

Another solution would be to surround with `{}`

Reply:
I prefer the variable to stay as local as possible, so `{ }` in the case statement. But how did it work before? Oh the limitation is only right after the `case`? Weird.

Reply:
ok, and weird indeed